config/_default/languages.yaml

```
@@ -24,3 +24,12 @@ zh-cn:
     languageISO: ZH
     languageTag: zh-CN
     dateFormat: "2006年1月2日"
+
+tr-tr:
+  languageName: Türkçe
+  contentDir: content/tr-tr
+  weight: 15
+  params:
+    languageISO: TR
+    languageTag: tr-TR
+    dateFormat: "2 Ocak 2006"
```

Currently, "Ocak" is always displayed. I think this needs to be:
```suggestion
    dateFormat: "2 January 2006"
```
It will still get translated.